service/src/test/java/tds/assessment/services/impl/AssessmentServiceImplTest.java

```diff
@@ -277,4 +292,139 @@ public void shouldReturnAssessmentWithItemProps() {
         assertThat(retCon2.getPropertyName()).isEqualTo(constraint2.getPropertyName());
         assertThat(retCon2.getPropertyValue()).isEqualTo(constraint2.getPropertyValue());
     }
+
+    @Test
+    public void shouldReturnAssessmentWithStrands() {
```

Comment:
Is there ever an instance where an assessment will not have strands?  If not, why have the strand specific stuff in its own test?  I would expect the `shouldReturnAnAssessment` to also include the strand logic if that is the expected result of an Assessment when calling `findAssessment`